kv/config.go

```diff
@@ -26,6 +26,11 @@ type Config struct {
 	IdleTimeout      time.Duration `mapstructure:"idle_timeout"`
 	IdleCheckFreq    time.Duration `mapstructure:"idle_check_freq"`
 	ReadOnly         bool          `mapstructure:"read_only"`
+	TLSConfig        TLSConfig     `mapstructure:"tls"`
+}
+
+type TLSConfig struct {
+	CaFile string `mapstructure:"ca_file"`
```

Comment:
Also, please, follow the standard RR configuration naming for the TLS options, example: https://github.com/roadrunner-server/grpc/blob/master/tests/configs/.rr-grpc-rq-tls-rootca.yaml#L20